### ISSUE TITLE
Cancel VRO pager duty

### ISSUE THREAD

**Issue by meganhicks** [department-of-veterans-affairs/abd-vro#3724]

**Comment by gabezurita** [comment]
This will be one of the last tasks and is blocked until all traffic stops on VRO (and migrations are complete — in prod). 

Message to vfs-platform-support when ready:
>  The VRO Platform is being decommissioned, and its PagerDuty service is no longer needed. However, I don’t seem to have the necessary permissions to disable it. Could you assist the VRO Team by shutting it down?
Here’s the link to the PagerDuty VRO service page:
https://dsva.pagerduty.com/service-directory/PUDS2VE

**Comment by bianca-rivera** [comment]
Blocked by [#3675](https://github.com/department-of-veterans-affairs/abd-vro/issues/3675)

**Comment by gabezurita** [comment]
Platform request here:
https://dsva.slack.com/archives/CBU0KDSB1/p1741030532921329

**Comment by gabezurita** [comment]
Disabled:
https://dsva.slack.com/archives/CBU0KDSB1/p1741036004087299?thread_ts=1741030532.921329&cid=CBU0KDSB1

**Comment by meganhicks** [comment]
@bianca-rivera are we okay to do this?

**Comment by bianca-rivera** [comment]
Yes, gave the ok yesterday
